standard_plus_issue/security/res_groups.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
@ventilooo The module category can be outside the noupdate.

Reply:
Can I have multiple <data> in one odoo tag ? Or do I need to have another file just for the module category ?